src/lib/components/Instances/InstanceTableRow.svelte

```diff
@@ -30,6 +30,12 @@
 		labelArea && autosize(labelArea);
 		predictionArea && autosize(predictionArea);
 	}
+
+	$effect(() => {
+		prediction?.then((pred) => {
+			metric = getMetric(prompt, localInstanceLabel || '', pred || '', project.metric_name);
```

Comment:
Does this run when localInstanceLabel changes? If so, we can remove the getMetric updates from below. Should check we're not re-running the metric function too many times

Reply:
No doesn't is inside a function only top level triggers.